aggregator_api/src/routes.rs

```diff
@@ -194,9 +204,15 @@ pub(super) async fn post_task<C: Clock>(
     })
     .await?;
 
-    Ok(Json(
-        TaskResp::try_from(task.as_ref()).map_err(|err| Error::Internal(err.to_string()))?,
-    ))
+    let mut task_resp =
+        TaskResp::try_from(task.as_ref()).map_err(|err| Error::Internal(err.to_string()))?;
+
+    // When creating a new task in the helper, we must put the unhashed aggregator auth token in the
+    // response so that divviup-api can later provide it to the leader, but the helper doesn't store
+    // the unhashed token and can't later provide it.
+    task_resp.aggregator_auth_token = aggregator_auth_token;
```

Comment:
Somewhere we ought to document the expected usage of the aggregator API for provisioning tasks, i.e. provision the helper first, then the leader using the generated auth token. Right now it's not clear that's the intended usage.

Doesn't have to be done in this PR.